eng/Helix.SingleTarget.targets

```diff
@@ -15,8 +15,12 @@
     <!-- Set TEST_DOTNET_ROOT to same as DOTNET_ROOT (which is provided by the dotnet cli correlation payload) -->
     <HelixWorkItemPreCommand Include="export TEST_DOTNET_ROOT=${DOTNET_ROOT}"
                              Condition="'$(IsHelixPosixShell)' == 'true'" />
+    <HelixWorkItemPreCommand Include="export LANG=en_US.UTF-8"
+                             Condition="'$(IsHelixPosixShell)' == 'true'" />
     <HelixWorkItemPreCommand Include="SET TEST_DOTNET_ROOT=%DOTNET_ROOT%"
                              Condition="'$(IsHelixPosixShell)' != 'true'" />
+    <HelixWorkItemPreCommand Include="SET LANG=en_US.UTF-8"
```

Comment:
This is not needed for Windows. It should only remain as a workaround for the incorrectly specified value for the Linux containers.